### ISSUE TITLE
parseFile() silently dies

### ISSUE THREAD

**Issue by epaaj** [smalot/pdfparser#198]
I installed pdfparser as it says on https://pdfparser.org/documentation and even used an example from there. But when it gets to ``` $parser->parseFile('document.pdf');``` it just dies without a word on my machine. It works fine on another machine so I know it can work.
It's when it tries to run Font::decodeOctal() from ElementString::parse() it fails. I can get logging until that method is run. Then there is nothing. It appears that Font::decodeOctal() is never called.

```php
<?php

// Include 'Composer' autoloader.
include 'vendor/autoload.php';

// Parse pdf file and build necessary objects.
$parser = new \Smalot\PdfParser\Parser();
$pdf    = $parser->parseFile('document.pdf');

?>
```

phpinfo():
https://gist.github.com/epaaj/b1749b49b24cc3ef257e4976cff0cc26

**Comment by nardnahcayaj** [comment]
Facing the same issue, can someone help on this.

**Comment by aironou** [comment]
When calling `Smalot\PdfParser\Parser()->parserFile()`, the return is silenced with `@`.
Debugging the code, I found the `Smalot\PdfParser\Object` class and that's the problem, because in PHP 7.2, Object as the class name is reserved.

Anyway, since version 0.11.0 of smalot/pdfparser, the problem has already been solved.

**Comment by smalot** [comment]
Fixed in the  v0.13.0 release.
Is it good for you ?

Thanks guys
